core/src/main/scala/org/apache/spark/scheduler/cluster/cook/CoarseCookSchedulerBackend.scala

```diff
@@ -115,20 +120,36 @@ private[spark] class CoarseCookSchedulerBackend(
   }
 
   var runningJobUUIDs = Set[UUID]()
+  var abortedJobIds = Set[UUID]()
+
+  private var jobLimitOption: Option[Int] = None
```

Comment:
Could you add a doc to explain what it is? Also, I would prefer remove `Option` from the var name.